FastSurferCNN/eval.py

```diff
@@ -88,6 +88,8 @@ def options_parse():
                         help='name under which segmentation will be saved. Default: aparc.DKTatlas+aseg.deep.mgz. '
                              'If a separate subfolder is desired (e.g. FS conform, add it to the name: '
                              'mri/aparc.DKTatlas+aseg.deep.mgz)')
+    parser.add_argument('--conf_name', '--conformed_name', dest='conformed_name', default='orig.mgz',
```

Comment:
We also save the original in orig/001.mgz, right? Maybe mention this here as well